packages/ses/src/lockdown-shim.js

```diff
@@ -184,13 +183,13 @@ export const repairIntrinsics = (
    */
   const seemsToBeLockedDown = () => {
     return (
-      Function.prototype.constructor !== Function &&
+      globalThis.Function.prototype.constructor !== globalThis.Function &&
       typeof globalThis.harden === 'function' &&
       typeof globalThis.lockdown === 'function' &&
-      Date.prototype.constructor !== Date &&
-      typeof Date.now === 'function' &&
+      globalThis.Date.prototype.constructor !== globalThis.Date &&
+      typeof globalThis.Date.now === 'function' &&
       // @ts-ignore
-      Object.is(Date.prototype.constructor.now(), NaN)
+      is(globalThis.Date.prototype.constructor.now(), NaN)
```

Comment:
These could probably be framed in terms of captured intrinsics, but I instead used `globalThis` pervasively to avoid changing the late-binding semantics.